### PR TITLE
Add exp-lookit-overview

### PR DESCRIPTION
**Pull request by samchrisinger**
# Purpose

Adds the exp-lookit-overview frame for the exp-player

Also some misc cleanup:
- DRY the jam-osf-jwt authenticator and handle 500s when trying to authenticate

### CODE REVIEW

File: exp-models/addon/authenticators/jam-osf-jwt.js [hunk 1]
```diff
@@ -23,32 +20,27 @@ export default Base.extend({
...
-            data: JSON.stringify({data: {
-                type: 'users',
-                attributes: {
-                    provider: 'osf',
-                    access_token
-                }
-            }})
-        }).then(function(res) {
-            res.data.attributes.accessToken = access_token;
+    authenticate(accessToken, _ /* expires */) {
```
Comment (abought) [inline exp-models/addon/authenticators/jam-osf-jwt.js]: Why is there an underscore in this call signature? Can the argument simply be omitted to ignore it instead?

As a bonus, that would get rid of another jshint error:
`exp-models/addon/authenticators/jam-osf-jwt.js: line 35, col 31, '_' is defined but never used.`

File: exp-models/addon/authenticators/jam-osf-jwt.js [hunk 2]
```diff
@@ -23,32 +20,27 @@ export default Base.extend({
                     access_token: accessToken
                 }
             }})
-        }).then(function(res) {
+        }).error(function(/*xhr, status, error*/) {
```
Comment (abought) [inline exp-models/addon/authenticators/jam-osf-jwt.js]: I may be going senile, but where is the `.error` method coming from?

JQuery [deferreds](https://api.jquery.com/category/deferred-object/), for example, use `.done` and `.fail`, while the A+ promises [specification](https://promisesaplus.com/) mainly seems to rely on the single `.then(done, fail)` method.

File: exp-models/addon/authenticators/jam-osf-jwt.js [hunk 3]
```diff
@@ -23,32 +20,27 @@ export default Base.extend({
...
+            this.transitionToRoute('login');
```
Comment (abought) [inline exp-models/addon/authenticators/jam-osf-jwt.js]: Is this transition necessary? It seems to me that experimenter always used to redirect to the login page when needed- was that an automatic behavior of ember simple auth? I'm hesitant to mix routing behavior into the generic logic module unless absolutely necessary; this couples the generic component to a specific application.

**Comment by abought** [comment]
Review complete; no major issues.

There may be a few template tweaks required, depending on outcome of conversations about schema for this component in:
https://github.com/CenterForOpenScience/experimenter/pull/43#discussion_r54432773

File: exp-models/addon/authenticators/jam-osf-jwt.js [hunk 4]
```diff
@@ -6,10 +6,7 @@ import config from 'ember-get-config';
 
 export default Base.extend({
     url: `${config.JAMDB.url}/v1/auth`,
-
-    restore(data) {
-        let accessToken = data.accessToken;
-
+    _post: function(accessToken) {
```
Comment (abought) [inline exp-models/addon/authenticators/jam-osf-jwt.js]: Al the Octopus approves of consolidating redundant code. :octopus:

File: exp-player/addon/templates/components/exp-lookit-overview.hbs [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<div class="col-md-12">
```
Comment (abought) [inline exp-player/addon/templates/components/exp-lookit-overview.hbs]: Markup nitpick, but don't bootstrap grids usually nest columns inside of rows? (not the other way around)

https://getbootstrap.com/css/#grid-example-basic

Comment (samchrisinger) [inline exp-player/addon/templates/components/exp-lookit-overview.hbs]: The new exp-player wrapper is now a "row".

**Pull request merged**